src/main/scala/mesosphere/marathon/storage/repository/GroupRepositoryImpl.scala

```diff
@@ -238,6 +238,37 @@ class StoredGroupRepositoryImpl[K, C, S](
       }
     }
 
+  @SuppressWarnings(Array("all")) // async/await
+  override def storeRootVersion(group: Group, updatedApps: Seq[AppDefinition]): Future[Done] =
+    async { // linter:ignore UnnecessaryElseBranch
+      val storedGroup = StoredGroup(group)
+      beforeStore match {
+        case Some(preStore) =>
+          await(preStore(storedGroup))
+        case _ =>
+      }
+      val promise = Promise[Group]()
```

Comment:
oops, that was for revert root which is not something we need for this one... will double check.